scvi/external/solo/_model.py

```diff
@@ -388,6 +388,47 @@ def auto_forward(module, x):
 
         return preds_df
 
+    @staticmethod
+    def setup_anndata(
+        adata: AnnData,
+        layer: Optional[str] = None,
+        copy: bool = False,
```

Comment:
@adamgayoso Do we need this for this model?

Reply:
I think copy should stay on all the methods, since its really just a general convenience function

Reply:
I wasnt asking if we need the copy arg, I was asking if we need the setup_anndata method